sunkit_instruments/suvi/io.py

```diff
@@ -0,0 +1,320 @@
+import os
+
+import h5py
+import numpy
+
+from astropy import units as u
+from astropy.io import fits
+from astropy.time import Time
+
+from sunkit_instruments.suvi import fix_L1b_header
```

Comment:
This function should be moved into this file as it is only called in the read function.

Reply:
Ok.